idojaras-quarterly-journal-of-the-hungarian-meteorological-service.csl

```diff
@@ -0,0 +1,235 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" and="text" demote-non-dropping-particle="never" default-locale="en-GB">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Id&#x151;j&#xE1;r&#xE1;s - Quarterly Journal of the Hungarian Meteorological Service</title>
+    <title-short>Id&#x151;j&#xE1;r&#xE1;s</title-short>
+    <id>http://www.zotero.org/styles/idojaras-quarterly-journal-of-the-hungarian-meteorological-service</id>
+    <link rel="self" href="http://www.zotero.org/styles/idojaras-quarterly-journal-of-the-hungarian-meteorological-service"/>
```

Comment:
did you base this style on an existing style? If so, which? Please include a `rel="template"` link to that style